### ISSUE TITLE
IGC repository connector build fails because of failed tests

### ISSUE THREAD

**Issue by lpalashevski** [odpi/egeria-connector-ibm-information-server#632]
Failed test ConnectorTest.verifySupportedTypes 

````
Error:  Tests run: 89, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 11.599 s <<< FAILURE! - in org.odpi.egeria.connectors.ibm.igc.repositoryconnector.ConnectorTest
Error:  org.odpi.egeria.connectors.ibm.igc.repositoryconnector.ConnectorTest.verifySupportedTypes  Time elapsed: 0.012 s  <<< FAILURE!
java.lang.AssertionError: expected [null] but found [OCFCheckedExceptionBase{reportedHTTPCode=404, reportingClassName='org.odpi.egeria.connectors.ibm.igc.repositoryconnector.IGCOMRSMetadataCollection', reportingActionDescription='verifyTypeDef', reportedErrorMessage='OMRS-IGC-REPOSITORY-404-001  The TypeDef "Person" has not been mapped to a pre-existing type in repository null', reportedErrorMessageId='OMRS-IGC-REPOSITORY-404-001 ', reportedErrorMessageParameters=[Person, null], reportedSystemAction='The system does not support the provided TypeDef, or it has not been mapped to a pre-existing type.', reportedUserAction='Raise an issue on the Egeria GitHub repository to see if it is feasible to map this TypeDef.', reportedCaughtException=null, reportedCaughtExceptionClassName='null', relatedProperties=null}]
	at org.odpi.egeria.connectors.ibm.igc.repositoryconnector.ConnectorTest.verifySupportedTypes(ConnectorTest.java:300)

[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    ConnectorTest.verifySupportedTypes:300 expected [null] but found [OCFCheckedExceptionBase{reportedHTTPCode=404, reportingClassName='org.odpi.egeria.connectors.ibm.igc.repositoryconnector.IGCOMRSMetadataCollection', reportingActionDescription='verifyTypeDef', reportedErrorMessage='OMRS-IGC-REPOSITORY-404-001  The TypeDef "Person" has not been mapped to a pre-existing type in repository null', reportedErrorMessageId='OMRS-IGC-REPOSITORY-404-001 ', reportedErrorMessageParameters=[Person, null], reportedSystemAction='The system does not support the provided TypeDef, or it has not been mapped to a pre-existing type.', reportedUserAction='Raise an issue on the Egeria GitHub repository to see if it is feasible to map this TypeDef.', reportedCaughtException=null, reportedCaughtExceptionClassName='null', relatedProperties=null}]
```

**Comment by lpalashevski** [comment]
Fuhrer investigation shows that Peron type is [extended](https://github.com/odpi/egeria/commit/d4ef55526c3f8559502aa27142b56dba33816958) with new properties (see screenshot). 

![image](https://user-images.githubusercontent.com/26764383/146950906-951740ca-9bd5-4e96-927c-d88bbbb11f32.png)

These updates will go in the next Open Metadata Types version 3.5 to be released in January but we already have dependency on the 3.5-snapshot version because of Data Engine connector interface updates.

@cmgrote I need help. 
What is the correct way to fix this? The test is failing the build now, should this be that strict or more like a warring? I am not sure what the potential implications are if we do not map all properties for given type (Person in this case).

**Comment by lpalashevski** [comment]
Rationale

Having strict control on mapped types/properties is very useful to prevent runtime errors. This allows us to verify typedefs as part of the dev process enforcing the check in unit test.

Solution 

- First part is to map the new properties to default values (i.e. null). This way we are already getting aware of what we can potentially map; (already implemented with 0f982f5b3efb23c87cc94df42e845a24c2cd9a16)
- Second part is to identify if these default values can get actual mappings (if the properties are actually available in the underlying technology in this case IGC). 

Fixed in https://github.com/odpi/egeria-connector-ibm-information-server/pull/628
Closing the issue.
